EntityFrameworkCore.Manipulation.Extensions.IntegrationTests/InsertIfNotExistTests.cs

```diff
@@ -91,5 +91,28 @@ public async Task InsertIfNotExistAsync_ShouldInsertAndReturnEntitiesWhichNotExi
             var actualEntities = await context.TestEntities.ToArrayAsync();
             actualEntities.Should().BeEquivalentTo(existingEntities.Concat(expectedEntitiesToBeInserted));
         }
+
+        // SQL server specific
+        [TestMethod]
+        public async Task InsertIfNotExistAsync_ShouldInsertDbNull_WhenObjectValueIsNull()
+        {
+            var expectedEntitiesToBeInserted = new[]
+            {
+                new TestEntity { Id = "1", StringTestValue = null, IntTestValue = 111, BoolTestValue = false, DateTimeTestValue = DateTime.UtcNow, LongTestValue = 15451455587546 },
+            };
+
+            using var context = await ContextFactory.GetDbContextAsync(DbProvider.SqlServer);
+
+            // Validate that the DB doesn't have any entities
+            (await context.TestEntities.CountAsync()).Should().Be(0);
```

Comment:
I think it would be a little cleaner to extract this out to a new variable, e.g.

```
var count = await context.TestEntities.CountAsync();
count.Should().Be(0);
```

Reply:
I'll fix this in the next update